packages/datetime/test/dateInputTests.tsx

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  */
 
+/**
+ * @fileoverview This component is DEPRECATED, and the code is frozen.
+ * All changes & bugfixes should be made to DateInput2 in the datetime2
+ * package instead.
+ */
+
+/* eslint-disable deprecation/deprecation, @blueprintjs/no-deprecated-components */
```

Comment:
I realize it is redundant to have both of these rules on as "error" violations, but I'm keeping them both so we can dogfood our own lint rule.